packages/apps-config/src/api/spec/dock.ts

```diff
@@ -0,0 +1,116 @@
+export default {
```

Comment:
im not a fan of hardcoding types here. can we import the docknetwork/sdk package types here and return them?

Reply:
Hardcoded as they for the dock-testnet chainspec, as we will have new chain spec, say poa-2 (test or main) or PoS (test or main) which can have different types, we will add new files and add new entries in the map in index.js